contracts/SingleBatchEdition.sol

```diff
@@ -135,15 +141,7 @@ contract SingleBatchEdition is
         override
         returns (uint256 lastTokenId)
     {
-        State memory _state = state;
-        if (msg.sender != _state.minter) {
-            revert Unauthorized();
-        }
-
-        if (totalSupply() > 0) {
-            revert SoldOut();
-        }
-
+        revertIfNotAuthorizedMinter();
```

Comment:
are we no longer checking if sold out?

Reply:
it's redundant now! see `testCanNotMintTwice()`